servarr/values.yaml

```diff
@@ -28,12 +28,50 @@ tags:
   # @default -- true
   # @section -- Tags
   music: true
+  # -- (bool) This tag will deploy: Scraparr
+  # @default false
+  # @section -- Tags
+  metrics: false
 
 metrics:
   # -- Anchor to set wether to deploy the export sidecar pods or not. Requires the Prometheus stack. Do not remove the `&metricsEnabled` anchor!
   # @section -- Metrics
   enabled: &metricsEnabled false
 
+scraparr: 
+  enabled: false
+  # -- Anchor to apiKey. As a single key is used for all services, same will be used for interacting with Sonarr, Radarr, Prowlarr
+  # @default 
+  # @section -- scarparr 
+  apiKey: *apikey
```

Comment:
This is incorrect. Scraparr requires the API per each services, e.g., `scraparr.config.<sonarr|prowlarr|radarr>.api_key`